src/extension/auth.js

```diff
@@ -211,3 +211,32 @@ export async function setAuthToken(
     await configureAuthAndCorsHeaders();
   }
 }
+
+/**
+ * Adds the sidekick version to the user agent for requests to the Admin API.
+ * @returns {Promise<void>}
+ */
+export async function updateUserAgent() {
+  const userAgent = `${navigator.userAgent} Sidekick/${chrome.runtime.getManifest().version}`;
```

Comment:
maybe `AEMSidekick`  ?

```suggestion
  const userAgent = `${navigator.userAgent} AEMSidekick/${chrome.runtime.getManifest().version}`;
```